packages/prover/scripts/generate_fixtures.ts

```diff
@@ -23,41 +23,104 @@ const networkURLs: Record<NETWORK, {beacon: string; rpc: string}> = {
 
 let idIndex = Math.floor(Math.random() * 1000000);
 
-async function rawEth(network: NETWORK, payload: Record<string, unknown>): Promise<Record<string, unknown>> {
-  return (await axios({url: networkURLs[network].rpc, method: "post", data: payload, responseType: "json"}))
-    .data as Record<string, unknown>;
+async function rawEth(
+  network: NETWORK,
+  method: string,
+  params: unknown[]
+): Promise<{payload: Record<string, unknown>; response: Record<string, unknown>}> {
+  const payload = {id: idIndex++, jsonrpc: "2.0", method, params};
+
+  const response = (
+    await axios({
```

Comment:
Why axios instead of native fetch? Will be supported by default in NodeJS too soon

Reply:
It was already used in other packages for e2e tests so used here for consistent pattern.